en/pocket_privacy_policy_eu.md

```diff
@@ -0,0 +1,124 @@
+
+## Collection of Information as Part of Using Pocket Service {: #passive }
+
+In addition to the information that you provide to us when you register for a user account, we collect information about the URLs, titles and content of the web pages and other information you save to Pocket. The types of information we collect includes your browser type, device type, time zone, language, and other information related to the manner in which you access the Pocket Technologies. If you are on a mobile device, we collect the advertising identifiers provided by Apple on iOS and by Google on Android. You can change this identifier in your device settings. We also collect information about your use of the Pocket Technologies so that we can provide our services. For example, as a part of providing Pocket’s syncing features, we sync information about the items that you save and view within Pocket so that your list, tags, scroll position, and other account and usage information may be synced across all of your devices.
+
+## Use of Cookies and Analytics {: #cookies }
+
+### Types and Purposes
+
+We use four types of cookies and analytics tools for the following purposes:
+
+- Necessary: These cookies are necessary for our website to function. They cannot be switched off. You can set your browser to block or alert you about these cookies, but this may cause parts of the site to not work properly. These cookies do not store any personally identifiable information and unless Personalized Advertising Cookies are enabled, ads are non-personalized.
+- Preferences: We place these cookies on your device to remember your account preferences and language settings. Disabling these cookies may break some features.
+- Analytics: We place these cookies on your device to help us understand how users engage with our website and product. For example, we use Google Analytics and other tools to understand which Pocket webpages are popular. We also use pixels in our emails in order to collect usage information.
+- Personalized Advertising: Some Pocket web pages have ads. With your consent, Pocket’s ad partners will place advertising cookies on your device to personalize the ads you see here and on other websites. If you play a YouTube video on Pocket, YouTube may place cookies on your device which can be used for personalization. Our partner for personalized ads on our syndicated article pages, Freestar, has its own [Privacy Policy](https://freestar.com/privacy-policy/).
+
+### Your Choices
+
+- Cookie Preference tool: By default, Pocket sets necessary cookies, and disables all other types of cookies. You can manage your settings for these cookies from this Cookie Preferences tool. [Learn more](https://help.getpocket.com/article/1163-pocket-website-cookies-faq).
```

Comment:
Besides analytics (which I assume is wanted), this is missing a reference to "preferences" compared to the US version. Is this wanted?

> Cookie Preference tool: By default, Pocket sets necessary, preference, and analytics cookies, and disables personalized advertising cookies.

Reply:
I'm not sure I understand the question. The EU version should say, "Cookie Preference tool: By default, Pocket sets necessary cookies, and disables all other types of cookies. You can manage your settings for these cookies from this Cookie Preferences tool."

Reply:
EU version:

> Cookie Preference tool: By default, Pocket sets necessary cookies, and disables all other types of cookies.

US version:

> Cookie Preference tool: By default, Pocket sets necessary, **preference, and analytics cookies,** and disables personalized advertising cookies.

I used bold on the additional text in the US version. It seems weird to me that we don't have cookies for **preferences** in EU?

Reply:
Current law is that all but necessary cookies must be off by default and require opt-in consent from users in EU.